logspout.yml

```diff
@@ -0,0 +1,18 @@
+version: '2'
+
+services:
+  logspout:
+    build: logspout/
+    volumes:
+      # security concern:
+      # https://raesene.github.io/blog/2016/03/06/The-Dangers-Of-Docker.sock/
+      # http://stackoverflow.com/questions/40844197
+      - /var/run/docker.sock:/var/run/docker.sock:ro
+    environment:
+      ROUTE_URIS: "logstash+tcp://logstash:5000"
```

Comment:
From the best I can gather, tcp connections will retry sending dropped metrics.

I did find this: 

`retry tcp errors and don't hang forever on failure`

--
 https://github.com/gliderlabs/logspout/blob/c2bafe3ea23a2372cf4d72bfc06785d819ad35d7/CHANGELOG.md#v3---2016-03-03